.github/workflows/examples.yaml

```diff
@@ -164,7 +164,7 @@ jobs:
         runs-on: ubuntu-latest
 
         container:
-            image: connectedhomeip/chip-build-efr32:0.4.8
+            image: connectedhomeip/chip-build:0.4.11
```

Comment:
did we lose the efr image?

Reply:
Yeah I messed up my `sed` command. Fixed.